app/scaffold-www/welcome-view/welcome.html

```diff
@@ -0,0 +1,33 @@
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+        <script src="../astro-client.js"></script>
+        <link href='https://fonts.googleapis.com/css?family=Lato:400,700,900' rel='stylesheet' type='text/css'>
```

Comment:
I'm not sure how hard it is to import it directly to the project, but I don't see the benefit of importing it into the Scaffold. 

I think this way, it is easy for developers to import other fonts by simply changing the URL. Since this view is only visible when we have internet, I don't think connectivity will be an issue either. What do you think @jeremywiebe?